ENGAGEHF/Bluetooth/Views/MeasurementRecordedView.swift

```diff
@@ -0,0 +1,123 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct MeasurementHeader: View {
+    @Environment(\.dismiss) var dismiss
+    @Binding var viewState: ViewState
+    
+    
+    var body: some View {
+        ZStack {
+            HStack {
+                Button(action: {
+                    dismiss()
+                }) {
+                    Text("DISMISS_BUTTON")
+                        .foregroundStyle(Color.accentColor)
+                }
+                    .buttonStyle(PlainButtonStyle())
+                    .padding(.trailing, 10)
+                    .disabled(viewState != .idle)
+                    .accessibilityLabel("Dismiss")
+                
+                Spacer()
+            }
+            
+            HStack {
+                Spacer()
+                
+                Text("Measurement Recorded")
+                    .font(.title2)
+                
+                Spacer()
+            }
+        }
+            .padding()
+    }
+}
+
+
+struct MeasurementLayer: View {
+    private var measurement = MeasurementManager.manager.newMeasurement
```

Comment:
I would suggest to access the measurement manager form the SwiftUI environment instead of using the shared instance to ensure that we could at some point more easily move away from the singleton. Getting this into place here would make this easier and is generally the way suggested for SwiftUI apps as it will also enforce that the view will be updated if the measurement manager changes.